CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ Non-backwards compatible changes
   Definitions that are sensitive to the behaviour of these lemmas, rather than
   just their existence, may need to be revised.
 
+* `RawIMonadT T` has been turned into a record type to allow the addition
+  of a lift field of type `∀ {M i j A} → RawIMonad M → M i j A → T M i j A`.
```

Comment:
This seems like a fairly big breaking in change in that users will have to add some new non-trivial code. Any chance you could write something brief giving them some hints about how to fix their code?